sapl/ip_restriction_middleware.py

```diff
@@ -0,0 +1,33 @@
+from django.http import HttpResponseForbidden
+
+
+class IpRestrictionMiddleware:
```

Comment:
Nit renomear a classe para algo como `EndpointRestrictionMiddleware` ou `EndpointGatekeeperMiddleware` ou algo do tipo. Acho que comunica melhor a intenção da classe. E afinal, estamos restringindo acesso a um endpoint. E no futuro podemos ter a necessidade de adicionar outros endpoints. 

```suggestion
class EndpointGatekeeperMiddleware:
```